src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,342 @@
+    }),
+    ui.Label("Corrective Labeling App - Malawi", {
+      fontSize: '24px',
+      fontWeight: 'bold',
+      //fontWeight: '120',
+    }),
+    ui.Label("", {
+      stretch: 'horizontal',
+    }),
+  ],
+  layout: ui.Panel.Layout.flow('horizontal'),
+});
+
+//Adding a Legend
+// set position of panel
+var legend = ui.Panel({
+  style: {
+    position: 'bottom-left',
+    padding: '8px 10px',
+    stretch: 'horizontal'
+  }
+});
+ 
+// Create legend title
+var legendTitle = ui.Label({
+  value: 'Legend',
+  style: {
+    fontWeight: 'bold',
+    position: 'bottom-left',
+    fontSize: '15px',
+    margin: '0 0 4px 0',
+    padding: '0'
+    }
+});
+
+// Creates and styles 1 row of the legend.
+var makeRow = function(color, name) {
+ 
+      // Create the label that is actually the colored box.
+      var colorBox = ui.Label({
+        style: {
+          backgroundColor: '#' + color,
+          // Use padding to give the box height and width.
+          padding: '8px',
+          margin: '0 0 4px 0'
+        }
+      });
+ 
+      // Create the label filled with the description text.
+      var description = ui.Label({
+        value: name,
+        style: {margin: '0 0 4px 6px'}
+      });
+ 
+      // return the panel
+      return ui.Panel({
+        widgets: [colorBox, description],
+        layout: ui.Panel.Layout.Flow('horizontal')
+      });
+};
+legend.add(legendTitle); 
+//  Palette with the colors
+var palette =['197228', 'FFFDCE'];
+ 
+// name of the legend
+var names = ['Crop','Non Crop',];
+ 
+//Add color and and names
+for (var i = 0; i < 2; i++) {
+  legend.add(makeRow(palette[i], names[i]));
+  }  
+
+//Add a description to the title
+var description =ui.Label({
+  value: 'This app allows users to add points to where the crop map is wrongly classified. Users can compare the cropland map on the left against the high resolution Sentinel 2 data on the right. The "Add a Marker" icon on the left panel can be selected to start creating the points. To delete a point, click the hand icon to select the specific point and hit the delete button. After picking all the points, users can click the "Get link for downloading points" button to generate a link that can be downloaded as a CSV file. A detailed instruction manual of the App can be found in the link below:' ,
+  style: {
+    stretch: 'horizontal',
+    position: 'top-center',
+    fontSize: '14px',
+    textAlign: 'center',
+    margin: '0 0 3px 0',
+    padding: '0',
+    color: '#3f3f3f',
+  }
+});
+
+var manual_link = ui.Label({
+  value: 'Detailed Instructions', 
+  style: {stretch: 'horizontal',
+    position: 'top-center',
+    fontSize: '15px',
+    //fontWeight: 'bold',
+    textAlign: 'center',
+    color: 'blue'}});
+    manual_link.setUrl('https://bit.ly/3EPT57D');
+
+var mainPanel = ui.Panel({
+  widgets: [
+    title,
+    description, manual_link, splitPanel, button, downloadUrl 
+  ],
+  style: {
+    width: '100%',
+    height: '100%',
+  }
```

Comment:
I don't think spreading this out over multiple lines is that helpful. It just adds more lines to the code and makes you scroll through more stuff. In my view it would be cleaner to write things like this in one line
```javascript
  style: {width: '100%', height: '100%'}
```

Reply:
The second squiggly bracket should be on the same line in this case